src/CommunityVoices/App/Api/Config/AccessControlList.json

```diff
@@ -133,6 +133,15 @@
         ["CommunityVoices\\App\\Api\\Controller\\Tag::getAllTag", {"allow": "guest"}],
         ["CommunityVoices\\App\\Api\\View\\Tag::getAllTag", {"allow": "guest"}],
 
+        ["CommunityVoices\\App\\Api\\Controller\\Tag::getTagUpload", {"allow": "admin"}],
+        ["CommunityVoices\\App\\Api\\View\\Tag::getTagUpload", {"allow": "admin"}],
+
+        ["CommunityVoices\\App\\Api\\Controller\\Tag::postTagUpload", {"allow": "guest"}],
+        ["CommunityVoices\\App\\Api\\View\\Tag::postTagUpload", {"allow": "guest"}],
```

Comment:
Why is there not a `postTagUpdate`?

Reply:
This was intentional, I figured that tags are simple enough that there should just be a delete and add new, so I was trying to avoid too much functionality. I can add an update if you would like

Reply:
Update seems to be added